distributed/client.py

```diff
@@ -454,6 +470,44 @@ def __str__(self):
 
     __repr__ = __str__
 
+    def _repr_html_(self):
+        if hasattr(self, '_loop_thread'):
+            info = sync(self.loop, self.scheduler.identity)
+        else:
+            info = False
+
+        text = ("<h3>Client</h3>\n"
+                "<ul>\n"
+                "  <li><b>Scheduler: </b>%s\n") % self.scheduler.address
+        if info and 'bokeh' in info['services']:
+            protocol, rest = self.scheduler.address.split('://')
+            port = info['services']['bokeh']
+            if protocol == 'inproc':
+                address = 'http://localhost:%d' % port
+            else:
+                host = rest.split(':')[0]
+                address = 'http://%s:%d' % (host, port)
+            text += "  <li><b>Dashboard: </b><a href='%(web)s' target='_blank'>%(web)s</a>\n" % {'web': address}
+
+        text += "</ul>\n\n"
+
+        if info:
+            workers = len(info['workers'])
+            cores = sum(w['ncores'] for w in info['workers'].values())
+            memory = sum(w['memory_limit'] for w in info['workers'].values())
+            memory = format_bytes(memory)
+            text2 = ("<h3>Cluster</h3>\n"
```

Comment:
Could this be a defined on `Cluster`, so that `client.cluster` has the same HTML repr for its info?